tests/test.rc

```diff
@@ -0,0 +1,38 @@
+/////////////////////////////////////////////////////////////////////////////
```

Comment:
The file header is not in accordance with the coding guidelines: https://wxwidgets.org/develop/coding-guidelines/#wxwin_header

I know it is incorrect (for historic reasons?) in the sample.rc as well, but I think we should always use the correct header for new files.

Also, can't we just include "../samples/sample.rc" from test.rc instead of duplicating and add only the cursor resource gere?

Reply:
I'll give that a try. I don't see why it wouldn't work.